flash_examples/integrations/labelstudio/text_classification.py

```diff
@@ -0,0 +1,34 @@
+import flash
+from flash.core.data.utils import download_data
+from flash.text import TextClassificationData, TextClassifier
+
+# 1. Create the DataModule
+download_data("https://label-studio-testdata.s3.us-east-2.amazonaws.com/lightning-flash/text_data.zip", "./data/")
+
+backbone = "prajjwal1/bert-medium"
+
+datamodule = TextClassificationData.from_labelstudio(
+    export_json="data/project.json",
+    val_split=0.8,
```

Comment:
val_split indicates what is the percentage going into val. Do you want 20 % in train, 80 % in val ?

Reply:
Fixed